docs/quickstart-cluster.rst

```diff
@@ -0,0 +1,262 @@
+.. _citus_quickstart:
+
+Citus Cluster Quick Start
+=========================
+
+In this guide we’ll create a Citus cluster with a coordinator node and two workers. Every node will have a secondary for failover. We’ll simulate failure in the coordinator and worker nodes and see how the system continues to function.
+
+For simplicity we’ll run all the nodes on a single machine, but in production the primary and secondary servers for each PostgreSQL node would run on independent machines.
+
+Before attempting this quickstart, you should try the :ref:`tutorial` for plain PostgreSQL nodes. The current guide relies on concepts explained there.
+
+Install pg_auto_failover and run a Monitor
+------------------------------------------
+
+pg_auto_failover requires a monitor node to keep track of global state and orchestrate actions on the other nodes. Follow the first two steps from the previous quickstart to get started:
+
+* :ref:`install`
+* :ref:`tutorial_run_monitor`
+
+.. note::
+
+  If you created a monitor *and registered nodes* in it as part of the previous quickstart guide, do not reuse the monitor. Create a new one with no nodes so that the cluster can be designated as type "citus" rather than type "postgres."
+
+Create a Citus Cluster
+----------------------
+
+pg_auto_failover can not only create nodes (as demonstrated in the previous quickstart), it can also configure them with the Citus extension. In this guide we'll use pg_auto_failover to create a coordinator node and two workers.
+
+First, install the Citus extension onto the system so pg_auto_failover can enable it on nodes:
+
+.. code-block:: bash
+
+  sudo yum install -y citus-rebalancer81_11
+
+Start a coordinator and its secondary. These commands will set up Citus on the node, and allow connections from the workers-to-be on the local network.
+
+.. code-block:: bash
+
+  pg_autoctl create coordinator  \
+    --auth trust              \
+    --ssl-self-signed         \
+    --pgdata ./coord_a        \
+    --pgport 6010             \
+    --nodename 127.0.0.1      \
```

Comment:
should be hostname I think
```
$ pg_autoctl create coordinator --pgdata /tmp/coord1 --monitor postgres://autoctl_node@localhost:6000/pg_auto_failover?sslmode=require --auth trust --ssl-self-signed --formation default --pgport 6001 --nodename localhost
coordinator: unrecognized option '--nodename'
```

Reply:
Reading this Citus HA tutorial now, after several years have passed, I'm inclined to do it all over again using docker compose this time. This shows better what instance runs on which node, uses a local network with hostnames, and we might be able to do things such as the following:

```
$ docker-compose up --scale coord=2 --scale worker=6
```

At the moment that would work only because the default strategy on the monitor is to assign two nodes per group, and then switch to another group. So we would have 3 workers nodes registered to the coordinator, and each one of them would have a secondary worker node available.

Still this would simplify the tutorial a lot and help convey the main concepts without spending too much time on implementation details.